input/pagecontent/index.md

```diff
@@ -118,6 +118,7 @@ This guide is divided into several pages which are listed at the top of each pag
   - [General Requirements](general-requirements.html): This page defines requirements common to all actors and profiles used in this guide including how CapabilityStatements are used to claim conformance. This page defines the expectations for mandatory and must support elements in AU Core Profiles.
 - [Guidance](guidance.html): This set of pages lists the guidance for this guide.
   - [General Guidance](general-guidance.html): This page provides guidance on using the profiles defined in this guide.
+  - [Medicine Information](medicine-information.html): This page provides guidance on the use of medications related profiles defined in this guide.
```

Comment:
Change this slightly "This page provides guidance on constructing medications related resources."

Reply:
Fixed.